### DISCUSSION TITLE
edit table

### DISCUSSION THREAD

**Discussion by compareme** [dbeaver/dbeaver#11997] [Q&A]
why can't edit table？the menu is not found

![image](https://user-images.githubusercontent.com/30946394/104694167-bcfd2a00-5745-11eb-9bab-126664ec7dd9.png)

**Comment by kseniiaguzeeva** [comment]
Most probably Simple view is enabled. Please Select in DB Navigator context menu for the database Connection view> Advanced
![dbeaver_ltLYXRlyl3](https://user-images.githubusercontent.com/51405061/104695173-611eab00-571d-11eb-83d1-ea01c0648cd4.png)

**Comment by compareme** [comment]
![image](https://user-images.githubusercontent.com/30946394/104699558-e5892200-574d-11eb-94a0-31be018cd80b.png)
like this?but it's inoperative

**Comment by kseniiaguzeeva** [comment]
Do you have View Table in the table context menu? Click it to open the table in Object Editor and edit the table.
Here is additional info:
https://github.com/dbeaver/dbeaver/wiki/Database-Object-Editor
https://github.com/dbeaver/dbeaver/wiki/Data-Editor
https://github.com/dbeaver/dbeaver/wiki/Data-Viewing-and-Editing
If it doesn't work, please tell me your database and DBeaver version.

**Comment by compareme** [comment]
![image](https://user-images.githubusercontent.com/30946394/104704689-ec1a9800-5753-11eb-82c8-223c4cad84db.png)
my  DBeaver version is Enterprise 7.3.0,I used the community version before ,it also just support for view  or increase the field,can't edit ,I think it's the version impact,the fact proved that's not,no matter what version,it can't edit field of table all

**Comment by kseniiaguzeeva** [comment]
Please tell the database name. Maybe DBeaver doesn't support it.

**Comment by compareme** [comment]
clickhouse and mysql ,I think it's not related to database

**Comment by compareme** [comment]
I see the version of yours is Community  7.3.3,DBeaver's download website show the latest version is 7.3.2,Why?
![image](https://user-images.githubusercontent.com/30946394/104708642-eecbbc00-5758-11eb-81c1-8ec6b2e8fd44.png)

**Comment by uslss** [comment]
seems it was EA version that is available here https://dbeaver.io/files/ea/
